blocks/api/utils.js

```diff
@@ -25,10 +25,11 @@ export function isUnmodifiedDefaultBlock( block ) {
 	}
 
 	const newDefaultBlock = createBlock( defaultBlockName );
-	const attributeKeys = [
+	// Omit the layout attribute because it's a special attribute used for nested blocks.
```

Comment:
Could we generalize this with a filter so we can keep the logic for layout consolidated in `blocks/hooks/layout.js`?